dkc/core/templates/core/trees_by_file_count.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="">
+  <head>
+    <meta charset="utf-8">
+  </head>
+
+  <body>
+    <table>
+      <thead>
+        <tr>
+          <th>Tree ID</th>
+          <th>Name</th>
+          <th>File count</th>
+          <th>Size</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for tree in trees %}
+        <tr>
+          <td>{{ tree.id }}</td>
+          <td>{{ tree.name }}</td>
+          <td>{{ tree.num_files }}</td>
+          <td>{{ tree.size }}</td>
```

Comment:
Note that with your modified query, we no longer get human-readable sizes directly from the query.

Reply:
https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#filesizeformat

Reply:
Fixed.